index.js

```diff
@@ -1023,9 +1023,11 @@ function buildArray (location, code, name, key = null) {
 
   code += `
     var l = obj.length
-    var w = l - 1
+
     for (var i = 0; i < l; i++) {
-      if (i > 0) {
+      var jsonLastChar = json.slice(-1)
+
+      if (i > 0 && jsonLastChar != '[' && jsonLastChar != ',') {
```

Comment:
please use `!==`

Reply:
My bad, totally missed this, gonna fix it 🙏